drivers/hmis/app/graphql/mutations/split_household.rb

```diff
@@ -0,0 +1,69 @@
+###
+# Copyright 2016 - 2025 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module Mutations
+  class SplitHousehold < BaseMutation
```

Comment:
Prefer using `CleanBaseMutation` to skip the input wrapper thing - I missed that on join households. Going forward we should use CleanBaseMutation but it would be too much churn to retroactively updated all the BaseMutation mutations.